CHANGELOG/CHANGELOG-0.10.md

```diff
@@ -0,0 +1,217 @@
+# Release notes for v0.10.0
+
+# Changelog since v0.9.1
+
+## Changes by Kind
+
+### Feature
+
+- Adds mappings for PV access modes to new CSI access modes: `SINGLE_NODE_SINGLE_WRITER` and `SINGLE_NODE_MULTI_WRITER`. ([#89](https://github.com/kubernetes-csi/csi-lib-utils/pull/89), [@chrishenzie](https://github.com/chrishenzie))
+
+### Other (Cleanup or Flake)
+
+- Csi-lib-utils library package dependencies are updated to make use of latest kube version v1.21.0 and also v1.4.0 version of csi spec which include many bug fixes and enhancements, with this release. ([#86](https://github.com/kubernetes-csi/csi-lib-utils/pull/86), [@humblec](https://github.com/humblec))
+- Sidecars exit without output about goroutines when loosing the connection to the driver ([#81](https://github.com/kubernetes-csi/csi-lib-utils/pull/81), [@pohly](https://github.com/pohly))
```

Comment:
I would prefix this with "Fixes issue where"

Reply:
Done, also fixed typo "loosing"